### ISSUE TITLE
Bug - Accessibility - Information is lost when text is resized to 200%

### ISSUE THREAD

**Issue by gordcorp** [AusDTO/gov-au-ui-kit#473]
This issue was identified as part of the accessibility testing performed on the DTA website Jan 2017: https://drive.google.com/open?id=0B3SXZ9dr83v0Y2cwV21HdUZ1STQ

The issue is related to the site and local nav on the DTA website, and since we are using these elements from uikit I believe this issue should be addressed in uikit.

**[5] Information is lost when text is resized to 200%**
Type: WCAG failure – 1.4.4 Resize text Desktop and mobile
Platform: Desktop and mobile
Severity: Medium

When text is resized to 200%, the global and main navigation items are no longer visible.
Note that we perform text-only resize testing for Success Criterion 1.4.4 Resize text since browser resizing will never fail this criterion. The text should be able to be resize up to 200%

![image](https://cloud.githubusercontent.com/assets/2324569/23347980/556d0148-fcfa-11e6-8096-3b992836ca9e.png)
Image: Screenshot of homepage navigation when text-only size is increased to 170%

![image](https://cloud.githubusercontent.com/assets/2324569/23347987/61743f2e-fcfa-11e6-993c-dd108adef738.png)
Image: Screenshot of homepage navigation when text-only size is increased to 200%. Main navigation has disappeared.

![image](https://cloud.githubusercontent.com/assets/2324569/23347989/6b25a7ec-fcfa-11e6-9e68-e9034c53f270.png)
Image: Screenshot of content page with level 2 navigation, when text-only size is increased to 200%. Main navigation is cut off.

**Recommendation**
Ensure that text size can be increased to 200% without any loss of content.
